src/js/utils/process-json.js

```diff
@@ -133,18 +153,28 @@ function processNumOriginationsData( data, group ) {
   } );
 
   data.projectedDate = {};
-  data.projectedDate.timestamp = getProjectedTimestamp( data.adjusted );
+  const fileNameMatch = 'inquiry';
```

Comment:
So this is essentially here as a label for 'inquiry'? Since this value is constant and only used a few lines down, what do you think about using it directly and including a code comment with further explanation on L:158:

```js
// Look for the string "inquiry" as part of the data source filename.
if ( source && source.includes( 'inquiry' ) ) {
```

Reply:
Filename has changed to `inq_` prefix (added in demos here https://github.com/cfpb/cfpb-chart-builder/pull/133). These are the names pushed to AWS https://files.consumerfinance.gov/data/consumer-credit-trends/auto-loans/inq_data_AUT.json

Reply:
A cleaner approach overall seems like if we could add a projected value field inside the data itself so this isn't tied to filename and so opening the data you can see what the projected values are set to. @hillaryj is that an option?